web/src/main/java/org/openmrs/web/Listener.java

```diff
@@ -338,7 +339,7 @@ private String getContextPath(ServletContext servletContext) {
 	private void clearDWRFile(ServletContext servletContext) {
 		String realPath = servletContext.getRealPath("");
```

Comment:
`realPath` is a string which contains the correct file separators. You can have a look at the method/interface javadocs.

the change you made would not work (same as above explanation) but if we remove `absPath` and inline this with

```java
File dwrFile = Paths.get(realPath, "WEB-INF", "dwr-modules.xml").toFile();
```